en/micro-integrator/docs/references/connectors/csv-module/csv-module-config.md

```diff
@@ -0,0 +1,559 @@
+
+The following transformations can be performed with this module.
+
+
+## CSV to CSV transformation
+
+You can use the CSV to CSV transformation to convert a CSV payload into another CSV payload according to your requirements using the configurations given below.
+
+### Operation details
+
+<table>
+<thead>
+  <tr>
+    <th>Name</th>
+    <th>Parameter</th>
+    <th>Value</th>
+    <th>Description</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td>Header</td>
+    <td>headerPresent</td>
+    <td>Absent<br>Present</td>
+    <td>Specify whether the CSV input has a header row</td>
+  </tr>
+  <tr>
+    <td>Separator</td>
+    <td>valueSeparator</td>
+    <td>Default : "," (comma)</td>
+    <td>Specify the separator to use in the CSV input.<br>To use tab as the separator, use the value tab to this property. To use space, use the value space.</td>
+  </tr>
+  <tr>
+    <td>Skip Headers</td>
+    <td>skipHeader</td>
+    <td>true, false</td>
+    <td>This is available only if the value of the <code>headerPresent</code> property is set to <code>Present</code>. The default value is <code>false</code>.</td>
+  </tr>
+  <tr>
+    <td>Skip Data Rows</td>
+    <td>dataRowsToSkip</td>
+    <td></td>
+    <td>Specify the number of data rows to skip in the CSV. The default is 0.<br>- If headerPresent is Present, then data rows are the rows excluding the first row.<br>- If <code>headerPresent</code> is <code>Absent</code>, then data rows are the rows starting from the first row.<br></td>
+  </tr>
+  <tr>
+    <td>Order by Column</td>
+    <td>orderByColumn</td>
+    <td></td>
+    <td>Order the CSV content by values of the given column. If you want to specify the column by column index, provide the index of the column (Indexes are starting from 1). <br>To specify the column by column name, give the column name within double quotes (e.g., "name"). <br>Specifying the column by column name works only if the value of the <code>headerPresent</code> property is <code>Present</code>.</td>
+  </tr>
+  <tr>
+    <td>Sort Columns</td>
+    <td>columnOrdering</td>
+    <td>Ascending, Descending</td>
+    <td>This option is enabled if the <code>orderByColumn</code> has a value. This determines whether the CSV should be ordered ascendingly or descendingly according to the given column.<br>The default value is <code>Ascending</code>.</td>
+  </tr>
+  <tr>
+    <td>Skip Columns</td>
+    <td>columnsToSkip</td>
+    <td></td>
+    <td>Specify columns to skip from the CSV payload. You can specify the columns as comma-separated values. <br>This property supports more complex queries also, you can find full specifications <a href="https://github.com/wso2-extensions/mediation-csv-module/blob/master/docs/column_skipper_query.md" target="_blank" rel="noopener noreferrer">here</a>.</td>
```

Comment:
Is it okay to point this to the GitHub repos docs? Or do we need to move that part of documentation also to this docs-ei repo?